.gitignore

```diff
@@ -102,3 +102,4 @@ dist
 
 # TernJS port file
 .tern-port
+test.md
```

Comment:
- [ ] [OPTIONAL]  I kindly suggest adding the node_modules directory to the .gitignore (as it can be added by accident while setting up linters).